### PR TITLE
docs: update lambda tutorial with fixes

### PR DESCRIPTION
**Pull request by lct45**
@wcarlson5 found some bugs when trying to follow the lambda tutorial, so I addressed those.

Additionally, if you want to run through all parts of the tutorial, it makes sense to name the streams + topics different things instead of reusing the same name over and over

https://confluentinc.atlassian.net/browse/DOCS-8217
https://confluentinc.atlassian.net/browse/DOCS-8216

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: docs/how-to-guides/use-lambda-functions.md [hunk 1]
```diff
@@ -90,13 +90,13 @@ Insert some values into `stream1`.
 INSERT INTO stream1 (
   id, lambda_map
 ) VALUES (
-  3, MAP("hello":= 15, "goodbye":= -5)
+  3, MAP('hello':= 15, 'goodbye':= -5)
 );
 ```
 
-Query the output.
+Query the output. Make sure to set auto.offset.reset = earliest.
```
Comment (JimGalasyn) [inline docs/how-to-guides/use-lambda-functions.md:97]:
```suggestion
Query the output. Make sure to set `auto.offset.reset = earliest`.
```

File: docs/how-to-guides/use-lambda-functions.md [hunk 2]
```diff
@@ -112,74 +112,74 @@ Your output should resemble:
...
+INSERT INTO stream3 (
+  id, lambda_map
 ) VALUES (
-  1, MAP("first name":= 15, "middle":= 25, "last name":= 0, "alt name":= 33)
+  1, MAP('first name':= 15, 'middle':= 25, 'last name':= 0, 'alt name':= 33)
 );
 ```
 
-Query the output.
+Query the output. Make sure to set auto.offset.reset = earliest.
```
Comment (JimGalasyn) [inline docs/how-to-guides/use-lambda-functions.md:180]:
```suggestion
Query the output. Make sure to `set auto.offset.reset = earliest`.
```

File: docs/how-to-guides/use-lambda-functions.md [hunk 3]
```diff
@@ -194,33 +194,33 @@ Your output should resemble:
...
+INSERT INTO stream4 (
+  id, lambda_map
 ) VALUES (
-  1, MAP("Mary":= ARRAY[1.23, 3.65, 8.45], "Jose":= ARRAY[5.23, 1.65]})
+  1, MAP('Mary':= ARRAY[1.23, 3.65, 8.45], 'Jose':= ARRAY[5.23, 1.65]})
 );
 ```
 
-Query the output.
+Query the output. Make sure to set auto.offset.reset = earliest.
```
Comment (JimGalasyn) [inline docs/how-to-guides/use-lambda-functions.md:221]:
```suggestion
Query the output. Make sure to set `auto.offset.reset = earliest`.
```

**Comment by lct45** [comment]
@JimGalasyn just cherry-picked to 17 and 18 if you could give those a sanity check (👍 
I assume it also needs to be pint-merged to master?

**Comment by JimGalasyn** [comment]
@lct45 Yes to pint merge, and thank you for cherry-picking!

**Pull request merged**